vfit.1

```diff
@@ -0,0 +1,35 @@
+.TH vfit "1" "December 2021" "User Commands"
+.SH NAME
+vfit \- generate backwards-compatible, static instances of variable fonts
+.SH SYNOPSIS
+vfit [\-h] [\-o PATH] [\-f {woff,woff2}] [\-C] config source
+.SH DESCRIPTION
+VFIT allows you to generate custom, static
+instances of a variable font defined in a configuration file.
+Generate backwards\-compatible, static instances of variable fonts.
+.SH OPTIONS
+.PP
+.SS "Positional arguments:"
+.TP
+[\config]
+the metadata/style definition file
+.TP
+[\source]
+the font to generate instances of
```

Comment:
Looks like something is broken here? I'm not familiar with the `man` page syntax or else I'd fix it myself. Looks like this for me on Debian:
<img width="414" alt="image" src="https://user-images.githubusercontent.com/41548458/146392321-2f40094b-0337-43cf-8909-530dfe62b818.png">